main/createUiDefinition.json

```diff
@@ -34,18 +34,75 @@
         },
         "visible": true
       },
+      {
+        "name": "vmSize",
+        "type": "Microsoft.Compute.SizeSelector",
+        "label": "VM size",
+        "toolTip": "Select the size of VM.",
+        "recommendedSizes": [
+          "Standard_F4s"
+        ],
+        "constraints": {
+          "allowedSizes": [
+            "Standard_B2s",
+            "Standard_B2ms",
+            "Standard_B4ms",
+            "Standard_B8ms",
+            "Standard_B12ms",
+            "Standard_B16ms",
+            "Standard_B20ms",
+            "Standard_F4s",
+            "Standard_F8s"
+          ]
+        },
+        "visible": "true",
+        "osPlatform": "Linux"
+      },
+      {
+        "name": "injecting",
+        "type": "Microsoft.Common.OptionsGroup",
+        "label": "Injecting",
+        "defaultValue": "Jointoken",
+        "toolTip": "Choose the type of injection.",
+        "constraints": {
+          "allowedValues": [
+            {
+              "label": "Jointoken",
+              "value": "Jointoken"
+            },
+            {
+              "label": "Custom Data",
+              "value": "Custom Data"
+            }
+          ],
+          "required": true
+        },
+        "visible": true
+      },
+      {
+        "name": "jointoken",
```

Comment:
will both jointoken and custom-data will be visible to user?

Reply:
we have to hide this in final release . it is only for QA validation